tests/xapian_tests/search_indexes.py

```diff
@@ -82,6 +82,35 @@ def prepare_empty(self, obj):
         return ''
 
 
+class UUIDBlogSearchIndex(BlogSearchIndex):
```

Comment:
Same here; it would be helpful if this is focused on the relevant aspect of the change.